src/main/webapp/style.css

```diff
@@ -2,14 +2,54 @@
 *  #264653 - charcoal can be used for background  
 *  #2A9D8F - persian green can be used as a header color
 *  #E9C46A - orange yellow can be used for text color
-*  #E76F51 - orange can be used as buttons
+*  #e76f51- orange can be used as buttons
 */
 
 
 /** style rules for index.html **/
 body, html {
   height:100%;
   text-align: center;
+  background-color: #264653;
+}
+
+h1, h2, h3, h4 { 
+  color: #2a9d8f;
+  text-align: center;
+  font-family: 'Press Start 2P', cursive;
+}
+
+h1 {
+  font-size: 5.4rem;
+}
+
+h2 {
+  font-size: 2.4rem;
+}
+
+h4 {
+    font-size: .96rem;
+}
+
+p {
+  color: #e9c46a;
+  font-size: 1.4rem;
+  line-height: 2.6rem;
+  font-family: 'B612', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  opacity: 0.75;
+}
+
+a:hover{
+    opacity: 1.0;
+}
+
+.step-outline {
+  -webkit-text-stroke-width: 2px;
+  -webkit-text-stroke-color: #e76f51;
```

Comment:
I've found it to be really useful to use variables for things like hex codes and font sizes that get used a lot. This seems like a good use for those.